src/gcp/storage.ts

```diff
@@ -0,0 +1,256 @@
+import * as path from "path";
+import * as api from "../api";
+import { logger } from "../logger";
+import { FirebaseError } from "../error";
+
+/** Bucket Interface */
+interface BucketResponse {
```

Comment:
These fields are all guaranteed to be present?

Reply:
Both the public documentation (https://cloud.google.com/storage/docs/json_api/v1/buckets#resource-representations) and the proto definition show all these fields as non-optional